yarn.lock

```diff
@@ -1123,137 +1123,137 @@
 "@ethersproject-xdc/abi@file:vendor/@ethersproject-xdc/abi":
   version "5.7.0"
   dependencies:
-    "@ethersproject-xdc/address" "file:vendor/@ethersproject-xdc/address"
```

Comment:
Please revert these changes to the `yarn.lock` file that change the XDC dependencies

Reply:
reverted